internal/io/mqtt/client/client.go

```diff
@@ -69,10 +69,7 @@ func (conn *Connection) onConnect(_ pahoMqtt.Client) {
 	conn.connected.Store(true)
 	conn.logger.Infof("The connection to mqtt broker is established")
```

Comment:
Do not need to handle subscription now. Looks like we can remove conn.subscriptions and all codes that manipulate it.